src/messaging.js

```diff
@@ -0,0 +1,90 @@
+// Copyright (C) <2015> Intel Corporation
```

Comment:
2020

Reply:
This file is based on the original `peerserver.js`, so it should still be 2015?

Reply:
this is new file any way. Please set to 2020.

Reply:
Done.